borsh/Cargo.toml

```diff
@@ -27,7 +27,7 @@ required-features = ["std", "unstable__schema"]
 cfg_aliases = "0.1.0"
 
 [dependencies]
-borsh-derive = { path = "../borsh-derive", version = "1.0.0-alpha.5", optional = true }
+borsh-derive = { path = "../borsh-derive", version = "~1.0.0", optional = true }
```

Comment:
tilde requirement is used, as (otherwise) due to `unstable__schema`, after release of `borsh-derive` of `1.1.` or `1.2`, a `1.0` `borsh` will pull these potentially incompatible versions, resulting in weird compilation errors.